tensorflow/lite/CMakeLists.txt

```diff
@@ -415,6 +415,11 @@ target_link_libraries(tensorflow-lite
     ruy
     ${TFLITE_TARGET_DEPENDENCIES}
 )
+
+if (BUILD_SHARED_LIBS)
+  list(APPEND TFLITE_TARGET_PUBLIC_OPTIONS "TFL_SHARED_LIBRARY_BUILD")
```

Comment:
we already use TFL_COMPILE_LIBRARY to build windows dll.

Reply:
TFL_COMPILE_LIBRARY switch between dllexport and dllimport. Neither could be applied if static library build

Reply:
So when BUILD_SHARED_LIBS is used, what's the expected definition of TFL_CAPI_EXPORT?
Do we still need to check TFL_COMPILE_LIBRARY?

Reply:
Yes. TFL_CAPI_EXPORT resolves to dllexport if TFL_COMPILE_LIBRARY otherwise to dllimport.